standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -669,8 +669,10 @@ public OpenTxnsResponse openTxns(OpenTxnRequest rqst) throws MetaException {
     }
     List<PreparedStatement> insertPreparedStmts = null;
     TxnType txnType = rqst.isSetTxn_type() ? rqst.getTxn_type() : TxnType.DEFAULT;
+    boolean isReplReplayOperation = rqst.isSetReplPolicy() && txnType == TxnType.REPL_CREATED;
```

Comment:
isReplayedReplTxn?
when we  can have txnType == TxnType.REPL_CREATED but rqst.isSetReplPolicy() == false?

Reply:
We can't have this situation ever.

Reply:
Then why that check

Reply:
Post [HIVE-24881](https://github.com/apache/hive/pull/2068), We can have a situation in which txnType == REPL_CREATED and rqst.isSetReplPolicy == false.
So, we need to make a check for replPolicy as well.